docs/pages/abide.md

```diff
@@ -48,8 +48,8 @@ These input types create a text field: `text`, `date`, `datetime`, `datetime-loc
   </div>
   <div class="row">
     <div class="medium-6 columns">
-      <label>URL Pattern, not required, but throws error if it doesn't match the Regular Expression for a valid URL.
-        <input type="text" placeholder="http://foundation.zurb.com" pattern="url">
```

Comment:
We should probably still highlight the URL pattern in the docs - can we add website as an additional rather than a replacement?